packages/model-viewer/src/features/ar.ts

```diff
@@ -373,6 +373,15 @@ configuration or device capabilities');
       const objectURL = generateUsdz ? await this.prepareUSDZ() : this.iosSrc!;
       const modelUrl = new URL(objectURL, self.location.toString());
 
+      if(generateUsdz){
+        const location = self.location.toString();
+        const locationUrl = new URL(location);
+        const srcUrl =  new URL(this.src!, locationUrl);
+        if(srcUrl.hash){
+          modelUrl.hash = srcUrl.hash;
```

Comment:
What is the behavior if someone specifies `allowsContentScaling=1` and `ar-scale="fixed"`?

Reply:
In testing, it appears the last instruction wins. QuickLook doesn't complain if there are duplicate params appended to the url, but respects the last one added. In this case, the model-viewer `ar-scale` property is appended after copying the `src` hash. So in the event someone specifies `allowsContentScaling=1` and `ar-scale="fixed"` the model will be presented at fixed scale.

Reply:
That sounds like good behavior to me, thanks for checking.